packages/block-library/src/buttons/editor.scss

```diff
@@ -3,6 +3,17 @@
 	width: auto;
 }
 
+.wp-block-buttons.is-direction-vertical .wp-block.block-editor-block-list__block[data-type="core/button"] {
+	display: block;
+	width: max-content;
+	margin-left: 0;
+}
+
+.block-editor-block-list__block[data-type="core/buttons"][data-align="center"] .wp-block-buttons.is-direction-vertical .wp-block.block-editor-block-list__block[data-type="core/button"] {
```

Comment:
This doesn't seem to have any effect on the rebased branch, as I think the block wrappers have changed. I don't think this will be needed if the margin-right can be reset in the above rule.

Reply:
Removed, thank you! ❤️